modules/ROOT/pages/about-data-types.adoc

```diff
@@ -1,30 +1,62 @@
-= About Data Types (Flow Designer)
+= Define and Use Data Types in a Flow (Flow Designer)
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-To build an integration application, you need to know what data is available at which step of the process and how to access the data.
+:imagesdir: ../assets/images
 
-* Is the data a string or an array item?
-* Is the data within the structure of an object?
-* Which parent elements do I need to reference to reach the data?
+Each card in a flow requires input data that is of a certain type, and each card outputs data that is of a certain type. As you build your flows, you should specify the data types that your cards require as inputs and produce as outputs. Doing so helps you plan the flows you create, because you know what data type is required by each card in the flow.
 
-In the early stage of development, you define data types to answer these questions. You include an example or schema of the expected data that the app uses. Some formats require a schema. Provide a schema for a CSV or Java format in a RAML file.
+A _data type_ explains to Flow Designer the formats and structures of the data that are in inputs and outputs. There are two categories of data types that are supported in Flow Designer: simple and custom. See these topics for information about these categories of data types:
```

Comment:
Use active voice and reduce repetitive words (data types)
Flow Designer supports two categories of data type: simple and custom. The following topics covers information about these categories: